webservices/docs.py

```diff
@@ -17,14 +17,10 @@
 item details with the other endpoints. If you are interested in individual donors, check
 out contributor information in schedule_a.
 
-If you would like to use the FEC's API programatically, you can sign up for
-your own API key below. But, you can still try out our API without
-an API key by using the web interface below and making use of a DEMO_KEY.
-Signing up for an API key will enable you to place up to 1,000 calls an hour.
-Each call is limited to 100 results per page. You can email questions, comments or
-a request to get a key for 120 calls per minute to [APIinfo@fec.gov](mailto:apiinfo@fec.gov). You can also
-ask questions and discuss the data in the [FEC data Google Group](https://groups.google.com/forum/#!forum/fec-data).
-API changes will also be added to this group in advance of the change.
+<h2>Getting started with the openFEC API</h2>
```

Comment:
This new header just needs to be bolded text. @johnnyporkchops Does this look okay to you?

`<p>
<strong>Getting started with the openFEC API</strong>
</p>`

Reply:
@johnnyporkchops Feel free to "edit" my comment to see the code I attempted.